### PR TITLE
KG - Report Date Time Zone Incomplete Visit Report

### PR DESCRIPTION
**Pull request by kyle-glick**

### CODE REVIEW

File: lib/reports/incomplete_visit_report.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ class IncompleteVisitReport < Report
 
   def generate(document)
     document.update_attributes(content_type: 'text/csv', original_filename: "#{@params[:title]}.csv")
-    _24_hours_ago = DateTime.now.ago(24*60*60)
+    _24_hours_ago = Time.now.ago(24*60*60).utc
```
Comment (amcates) [inline lib/reports/incomplete_visit_report.rb]: 24.hours.ago.utc

File: app/models/procedure.rb [hunk 2]
```diff
@@ -41,7 +41,7 @@ class Procedure < ActiveRecord::Base
   # select Procedures that belong to an Appointment without a start date
   scope :belonging_to_unbegun_appt, -> { joins(:appointment).where('appointments.start_date IS NULL') }
   scope :completed_r_in_date_range, ->(start_date, end_date) {
-        where("procedures.completed_date is not NULL AND DATE(procedures.completed_date) between ? AND ? AND billing_type = ?", start_date.to_date, end_date.to_date, "research_billing_qty")}
+        where("procedures.completed_date is not NULL AND procedures.completed_date between ? AND ? AND billing_type = ?", start_date.to_date, end_date.to_date, "research_billing_qty")}
```
Comment (amcates) [inline app/models/procedure.rb]: to_date

File: app/models/fulfillment.rb [hunk 3]
```diff
@@ -20,10 +20,10 @@ class Fulfillment < ActiveRecord::Base
   after_create :update_line_item_name
 
   scope :fulfilled_in_date_range, ->(start_date, end_date) {
-        where("fulfilled_at is not NULL AND DATE(fulfilled_at) between ? AND ?", start_date.to_date, end_date.to_date)}
+        where("fulfilled_at is not NULL AND fulfilled_at between ? AND ?", start_date.to_date, end_date.to_date)}
```
Comment (amcates) [inline app/models/fulfillment.rb]: to_date

**Comment by kyle-glick** [comment]
@amcates 
An alternative to the + 23.hours + 59.minutes + 59.seconds would be + 1.day - 1.second. Would that work better?

**Comment by amcates** [comment]
I think either works fine.  Ask them team and go with the consensus. 

Andrew

**Comment by kyle-glick** [comment]
@amcates 
Even better, we realized theres a Time.tomorrow method, so we did .tomorrow.utc - 1.second

**Comment by sparc-request** [comment]
Nice

On Feb 11, 2016, at 11:30 AM, kyle-glick <notifications@github.com<mailto:notifications@github.com>> wrote:




@amcates<https://github.com/amcates>
Even better, we realized theres a Time.tomorrow method, so we did .tomorrow.utc - 1.second

**Pull request merged**